README.md

```diff
@@ -1,63 +1,79 @@
-their own app superpeer. This would be useful for example, if
-developers wanted to visualize data coming from the mesh using a
-traditional website, or create some interaction between the web and the
-mesh.
-
-In this case, they can use the developer
-library and set the superpeer themselves with the mm.setSuperPeer
-function (and pass the IP address or hostname of the superpeer).
-
-In the longer term, the production library will not support this
-function, however, developers will be able to determine the MeshID of
-their superpeer(s) and use those to let the Mesh determine the best way
-to reach the superpeer(s).
-
-Don't forget when you clone the repo to update your credentials from 
-[https://developer.rightmesh.io] in the [build.gradle](build.gradle) 
-file.
-
-# Setting up
-If you want to run the parity client directly, run the `configure.sh`
-script. This will install parity from .deb (along with all dependencies)
-and start it up.
+# RightMesh Superpeer
 
-Otherwise you can uncomment the last few lines in the 
-build.gradle script to run parity in vagrant. If you wish to run it
-in vagrant, you'll need to install vagrant and virtualbox. This will
-install the parity wallet within the vm so you don't accidentally
-clobber your local wallet.
+This minimalist RightMesh "Superpeer" implementation provides two functionalities: enabling the forwarding of data packets from one geographically separate mesh to another, and facilitating transactions involving RightMesh Tokens. The second functionality includes recording transactions to the Ethereum blockchain.
+
+## Running the Superpeer
+
+Superpeer is a Gradle application - it can be built with `gradlew build`, and a binary can be generated with `gradlew installDist`.
+
+If run without arguments Superpeer polls STDIN for input - typing `exit` will shut down RightMesh and stop the application. Unless you are developing/debugging, you will likely want to run Superpeer with the `-h | --headless` flag, which doesn't poll for input and responds to SIGINT signals (e.g. can be killed cleanly with `Ctrl+C` or task managers).
+
+## Connecting to your Superpeer
+
+By default RightMesh devices with internet connections will connect to a Superpeer operated by RightMesh at `research.rightmesh.io`. In the future the goal is to have a network of Superpeers, with Superpeers implemented and operated by both RightMesh and community members.
+
+If you would like to have devices use your Superpeer instance for testing, you can hard-code its address into a RightMesh application to make it use your instance instead of the production network. You can do this by passing the address as a parameter to a special MeshManager constructor that is available in the developer version of the library. Note that this will never be an option in production - all RightMesh applications will use the same network, so that the owner of the device can specify their preferences for where/when/how to connect to Superpeers.
```

Comment:
We should probably be consistent with terminology - do we want to call it release or production? androids terminology for the target itself is release.

Reply:
Now that I think about it I wonder if borrowing terminology from the crypto space would make more sense, something like the "RightMesh Mainnet".

"Release" is too ambiguous I think, like a release is a thing but a release can be outdated, it doesn't refer to the _latest_ release.

"Production" tends to imply the thing that is being put in front of customers but maybe doesn't get across what I mean here. I feel like beta releases of some software would still use the "Mainnet", even though they aren't "Production."

Reply:
I agree with using the term `Mainnet`, I think it makes more sense